x-pack/plugins/enterprise_search/server/routes/workplace_search/sources.ts

```diff
@@ -749,26 +729,6 @@ export function registerOrgSourceReindexJobRoute({
   );
 }
 
-export function registerOrgSourceReindexJobStatusRoute({
-  router,
-  enterpriseSearchRequestHandler,
-}: RouteDependencies) {
-  router.get(
-    {
-      path: '/api/workplace_search/org/sources/{sourceId}/reindex_job/{jobId}/status',
```

Comment:
We no longer need to poll for reindexing job statuses now that they complete immediately, hence these API route removals.